packages/app/src/ServiceDashboardPage.tsx

```diff
@@ -269,7 +288,230 @@ export default function ServiceDashboardPage() {
+                                where: scopeWhereQuery('span.kind:"server"'),
+                                groupBy: ['span_name'],
+                                numberFormat: {
+                                  factor: 1,
+                                  output: 'number',
+                                  mantissa: 2,
+                                  thousandSeparated: true,
+                                  average: false,
+                                  decimalBytes: false,
+                                  unit: 'ms',
+                                },
+                              },
+                              {
+                                displayName: 'Median',
+                                table: 'logs',
+                                type: 'table',
+                                aggFn: 'p50',
+                                field: 'duration',
+                                where: scopeWhereQuery('span.kind:"server"'),
+                                groupBy: ['span_name'],
+                                numberFormat: {
+                                  factor: 1,
+                                  output: 'number',
+                                  mantissa: 2,
+                                  thousandSeparated: true,
+                                  average: false,
+                                  decimalBytes: false,
+                                  unit: 'ms',
+                                },
+                              },
+                              {
+                                displayName: 'Total',
+                                table: 'logs',
+                                type: 'table',
+                                aggFn: 'sum',
+                                field: 'duration',
+                                where: scopeWhereQuery('span.kind:"server"'),
+                                groupBy: ['span_name'],
+                                sortOrder: 'desc',
+                              },
+                              {
+                                displayName: 'Errors',
+                                table: 'logs',
+                                type: 'table',
+                                aggFn: 'count',
+                                field: '',
+                                where: scopeWhereQuery(
+                                  'span.kind:"server" level:"error"',
+                                ),
+                                groupBy: ['span_name'],
+                              },
+                            ],
+                            seriesReturnType: 'column',
+                          }}
+                        />
+                      </Card.Section>
+                    </Card>
+                  </Grid.Col>
+                  <Grid.Col span={12}>
+                    <Card p="md">
+                      <Card.Section p="md" py="xs" withBorder>
+                        Debug
```

Comment:
Should this Debug section still be here?

Reply:
this page is behind a flag so it's alright afaik